jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.yaml

```diff
@@ -277,7 +304,7 @@
             source scripts/qa_crowbarsetup.sh
             ./sbd_prepare
 
-            timeout --signal=ALRM 60m bash -x -c "source scripts/qa_crowbarsetup.sh ; onadmin_runlist batch;"
+            timeout --signal=ALRM 60m bash -x -c "source scripts/qa_crowbarsetup.sh ; crowbar batch --timeout 2400 build scenario.yml;"
```

Comment:
`crowbar batch --timeout 2400 build scenario.yml;` should be enough on this line as you avoid using the `onadmin_batch` function